src/formatDate.js

```diff
@@ -50,7 +50,32 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = date.split(fromFormat.pop());
+  const separator = toFormat.pop();
+  const objFromRormat = {};
```

Comment:
Consider renaming `objFromRormat` to something more descriptive like `dateObject` or `formattedDate`. 📝